### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by JayP718** [wa0x6e/cal-heatmap#299] [Q&A] [answered]
Hi there @wa0x6e !

Firstly thank you for such a great tool. I was just wondering how I can go about accesing the code behind the examples in version 4. I know you are still working on the documentation but was hoping I can dig through the code to get a better understanding on how to configure it.

Thank you!

J

**Comment by wa0x6e** [comment]
Good idea, will add them later via jshint on final build release

**Comment by wa0x6e** [comment, marked as answer]
Examples are now displayed via a live code editor, thanks to React and the new doc engine

**Reply by JayP718** [reply]
Thank you!
